### ISSUE TITLE
feature request: using kindle-dash to add screen for RaspiBlitz

### ISSUE THREAD

**Issue by woeisme** [raspiblitz/raspiblitz#3300]
many of us are running our RaspiBlitz headless but would like to add screens. 
Sometimes acquiring an old kindle is easier than a full screen display.
[This project](https://github.com/pascalw/kindle-dash) repurposes old (jailbroken!) kindles as a display device, having the kindle device itself fetch data via curl at regular intervals. 

I think it would make a nice integration to the RaspiBlitz project.

I have a spare PW kindle I may begin tinkering with and update this ticket as needed

**Comment by NilByte** [comment]
Check this one.

https://github.com/rootzoll/raspiblitz/blob/v1.8/home.admin/config.scripts/bonus.kindle-display.sh

**Comment by openoms** [comment]
Yes, already using https://github.com/dennisreimann/kindle-display and as above can connect to the blitz.

Modifications to the server (in JS) are welcome to show any other info.

**Comment by perduewu-ops** [comment]
For a newer host-rendered Kindle display pattern, this may be useful: https://github.com/perduewu-ops/kindle-side-card
